src/main/java/com/moviecat/services/GenreServices.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Not sure why we need this interface. Kindly explain.

Reply:
Interfaces are necessary to achieve `full abstraction in the object-oriented environment`.